collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:background="@drawable/project_settings_dialog_background"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/close_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_close"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/projects"
+        style="@style/TextAppearance.Collect.Subtitle1"
+        app:layout_constraintBottom_toBottomOf="@+id/close_icon"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/close_icon" />
+
+    <com.google.android.material.button.MaterialButton
```

Comment:
I think we should avoid deviating from our theme on these buttons. @lognaturel I'm guessing you were just using placeholders from the default material theme in your mockups? If so, I think this can just use the `Widget.MaterialComponents.Button.OutlinedButton` style (again worth double-checking) which means we won't need `testAllCaps`, `cornerRadius`, `strokeColor`, `strokeWidth`, `backgroundTint` or `testColor`.

Reply:
Right, I did just copy and paste the material theme and didn't intend for those to be prescriptive, sorry about not making that clear.

Using our standard outline buttons might be fine. My guess is that they will still look really big and out of place. I suggest either leaving it as-is or removing all the custom styles for now and refining once all the pieces are in place. I've also put a couple of other options [in the Figma file](https://www.figma.com/file/nDsMAYQDIflyl7rznEhwnN/ODK-Collect-Multiple-Project-Support-(v1.31)?node-id=0%3A1) to consider. Thinking about it again, it may make more sense to use list items rather than buttons.

Reply:
I removed those custom properties and used `style="@style/Widget.MaterialComponents.Button.OutlinedButton"` for now. It looks good and we can get back to it later if needed.